.travis/pinned-deps.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```

Comment:
Somehow tests are failing due to removal of this file.

Reply:
Fixed